liferay-gradle/blade.reluctant.service.reference.override/blade.reluctant.service.reference.custom/build.gradle

```diff
@@ -0,0 +1,37 @@
+repositories {
+flatDir {
+	   dirs 'lib'
```

Comment:
Lets remove the lib file from this project and use a `compile project(":blade.reluctant.service.reference.old")` instead, then we can delete the jar file.